openapi-generator-annotations/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/gibahjoe/openapi-generator-dart
 
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.17.0 <3.0.0'
```

Comment:
This allows for usage of `ConstructorName({super.field})`

Reply:
Let's keep backward compatibility to a max at least till the next major release.